Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.view.html

```diff
@@ -233,23 +233,23 @@ <h4 data-element-type="dpaMainDetailHeader">{{vm.headerName}}</h4>
                 <table id="subDpTable" class="table table-striped table-bordered">
                     <thead>
                         <tr>
-                            <th class="element-selection-list_header--padding">
+                            <th width="38%" class="element-selection-list_header--padding">
                                 <p class="small">Navn</p>
                             </th>
-                            <th class="element-selection-list_header--padding">
+                            <th width="7%" class="element-selection-list_header--padding">
                                 <p class="small">CVR</p>
                             </th>
-                            <th class="element-selection-list_header--padding">
+                            <th width="25%" class="element-selection-list_header--padding">
                                 <p class="small">Overførselsgrundlag</p>
                             </th>
-                            <th class="element-selection-list_header--padding">
+                            <th width="15%" class="element-selection-list_header--padding">
                                 <p class="small">Overførsel til usikkert 3. land</p>
                             </th>
-                            <th class="element-selection-list_header--padding noRightBorder">
+                            <th width="20%" class="element-selection-list_header--padding noRightBorder">
                                 <p class="small">Land der overføres til</p>
                             </th>
-                            <th class="element-selection-list_header--padding noLeftBorder noRightBorder"></th>
-                            <th class="element-selection-list_header--padding noLeftBorder"></th>
+                            <th width="5%" class="element-selection-list_header--padding noLeftBorder noRightBorder"></th>
+                            <th width="5%" class="element-selection-list_header--padding noLeftBorder"></th>
```

Comment:
@a-naskret 
you specified **115%**

Reply:
@a-naskret I fixed you PR now, but really.. I should not have to for stuff like this :'(